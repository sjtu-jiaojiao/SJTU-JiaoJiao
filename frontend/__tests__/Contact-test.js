import React from 'react';
import ContactScreen from '../src/Views/Contact/Contact';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<ContactScreen/>).toJSON();
    expect(tree).toMatchSnapshot();
})
