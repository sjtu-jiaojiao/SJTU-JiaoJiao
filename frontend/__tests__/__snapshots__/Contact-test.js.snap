// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text>
    这是信息界面
  </Text>
</View>
`;
